docs/benchmarks/rabi_oscillations.ipynb

```diff
@@ -0,0 +1,488 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            v = hashlib.sha256(str(q).encode()).digest()[0] / 256
</pre><p>lol why are we doing this hashing? can't we just put a number</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/ReCirq/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I'm not sure. Maybe the original author intended it to hide the actual value of $v$</p>